osu.Framework/Graphics/Batches/VertexBatch.cs

```diff
@@ -6,17 +6,15 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
-using osu.Framework.Graphics.OpenGL.Buffers;
-using osu.Framework.Graphics.OpenGL.Vertices;
 using osu.Framework.Graphics.Rendering;
 using osu.Framework.Statistics;
 
 namespace osu.Framework.Graphics.Batches
 {
-    internal abstract class VertexBatch<T> : IVertexBatch<T>
-        where T : struct, IEquatable<T>, IVertex
+    public abstract class VertexBatch<T> : IVertexBatch<T>
```

Comment:
Any reason for public?

Reply:
Sounds like it, will change back.

Reply:
Can be done separately. All OpenGL/Veldrid classes should be made `internal` at some point.